core/src/sumeragi/message.rs

```diff
@@ -89,14 +87,12 @@ pub enum Message {
 #[non_exhaustive]
 pub struct BlockCreated {
```

Comment:
We were talking about revalidation of `CommittedBlock`, but the same applies for the `PendingBlock`.
We can't trust that we receive validated block and must revalidate it.

Reply:
Is that not what sumeragi is for? ValidatingPeers + ProxyTail all validate the block and the leader does not because it created it? Maybe there's a case for the ObservingPeers but they are supposed to receive all their blocks through block sync. (I know this is currently not the case but I have a ticket for this to do.)

Reply:
Yeah, but validating peers and proxy tail receive `BlockCreated` message and we want to make sure that peer don't accidentally forget to validate it.

Reply:
I have now added this.